amplify/backend/function/updateContributorsWithReputation/src/utils.js

```diff
@@ -281,6 +282,29 @@ const updateReputationInDomain = async ({
   );
 };
 
+const calculatePercentageReputation = (userReputation, totalReputation) => {
```

Comment:
I took this (and slightly edited it to work in the lambda) from `src/utils/reputation.ts` so that it was being calculated in a uniform way across the app. Before, there were minor differences leading to slightly different percentages being shown.